test/model/message_test.dart

```diff
@@ -151,8 +155,11 @@ void main() {
 
       connection.prepare(
         json: UpdateMessageResult().toJson(), delay: Duration(seconds: 1));
-      store.editMessage(messageId: message.id, newContent: 'new content');
-      checkRequest(message.id, 'new content');
+      store.editMessage(messageId: message.id,
+        originalRawContent: 'old content', newContent: 'new content');
+      checkRequest(message.id,
+        prevContentSha256: '34a780ad578b997db55b260beb60b501f3e04d30ba1a51fcf43cd8dd1241780d',
```

Comment:
Presumably this is the hash of the string "old content"? Probably clearest to say it that way directly. (The computing of the hash can get centralized into this `checkRequest` helper.)